### PR TITLE
[FIX] Replace openerp with odoo

### PR DESCRIPTION
**Pull request by App-247**

### CODE REVIEW

**Comment by lasley** [comment]
Thanks for the submission @App-247 

@obulkin - can you please review and see to it that the changes also make their way to OCA?

File: medical/models/medical_patient.py [hunk 1]
```diff
@@ -36,12 +36,19 @@ class MedicalPatient(models.Model):
         compute='_compute_is_deceased',
     )
     marital_status = fields.Selection([
-        ('s', 'Single'),
-        ('m', 'Married'),
-        ('w', 'Widowed'),
-        ('d', 'Divorced'),
-        ('x', 'Separated'),
-        ('z', 'law marriage'),
+        ('S', 'Never Married'),
```
Comment (obulkin) [inline medical/models/medical_patient.py]: Please leave all of the old selection keys in place for backward compatibility. The values can be modified as needed for greater alignment with FHIR, and new options are definitely welcome.

File: medical_medicament_attribute/views/medical_medicament_attribute_type_view.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <!--
     Copyright (C) 2016 LasLabs [https://laslabs.com]
     @author Dave Lasley <dave@laslabs.com>
-    The license is in the file __openerp__.py
+    The license is in the file __odoo__.py
```
Comment (obulkin) [inline medical_medicament_attribute/views/medical_medicament_attribute_type_view.xml]: The v10 equivalent of ``__openerp__.py`` is ``__manifest__.py``. However, if you're gonna update these at all, I would instead suggest just switching over to the latest OCA copyright format, which can be found [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/data/model_name_data.xml#L2). The module is licensed under LGPL-3.0.

File: medical_physician/views/medical_specialty_view.xml [hunk 3]
```diff
@@ -77,4 +77,4 @@
         </record>
 
     </data>
-</openerp>
\ No newline at end of file
+</odoo>
```
Comment (obulkin) [inline medical_physician/views/medical_specialty_view.xml]: There needs to be a newline character here.

File: medical_physician/wizard/medical_physician_unavailable_view.xml [hunk 4]
```diff
@@ -39,4 +39,4 @@
         </record>
 
     </data>
-</openerp>
\ No newline at end of file
+</odoo>
```
Comment (obulkin) [inline medical_physician/wizard/medical_physician_unavailable_view.xml]: Ditto on the newline.

**Review by obulkin — CHANGES_REQUESTED** [review]
Thanks for the submission, @App-247. I've made a few minor comments in line.

Also, OCA guidelines require a separate commit per affected module, so we may not be able to preserve attribution when porting this over to the OCA repo unless you rebase your branch accordingly.

**Comment by lasley** [comment]
Hey @App-247 - I just looked in the OCA instance and I can't find a [CLA](https://odoo-community.org/page/cla) for you. Is this your first contribution to an OCA project?

**Comment by App-247** [comment]
Yes. I had forgotten. Right now I do it.

**Comment by lasley** [comment]
Hi @App-247 - there was a DMCA in this repo and I'm having to close this PR because it's based on offending code.

We're moving to the OCA rep, so it would be best to just submit the change there.

**Pull request closed without merging**